src/setupTests.js

```diff
@@ -10,14 +11,16 @@ process.env.DEBUG_PRINT_LIMIT = 15000
 // it wouldn't work anyway
 jest.mock('./test/server', () => {})
 
-// instead, we'll use the same handlers we use for the service worker
-// as a mock fetch
+const mockServer = setupServer(...handlers)
+
 beforeEach(() => {
-  jest.spyOn(window, 'fetch').mockImplementation(fetchMock)
+  // enable API mocking in test runs using the same request handlers
+  // as for the client-side mocking.
+  mockServer.listen()
```

Comment:
Is there a reason to enable/restore mocks on before each test? 
I think it'd be much more performant to do that before/after all.